client/views/omnichannel/currentChats/CurrentChatsRoute.js

```diff
@@ -67,9 +67,10 @@ const useQuery = ({ guest, servedBy, department, status, from, to, tags, customF
 		...itemsPerPage && { count: itemsPerPage },
 		...current && { offset: current },
 	};
-
 	if (from && to) {
-		query.createdAt = JSON.stringify({ start: from, end: to });
+		query.createdAt = JSON.stringify({
```

Comment:
Are you sure these changes will fix the issue?
When I set just the first date, nothing happens.